protocol-designer/src/resources/types.ts

```diff
@@ -0,0 +1,3 @@
+export type PrefixType = 'aspirate' | 'dispense' | 'mix'
```

Comment:
looks good but maybe something more descriptive for the prefix like 
```suggestion
export type MoveLiquidPrefixType = 'aspirate' | 'dispense' | 'mix'
```